runtime/doc/lua.txt

```diff
@@ -969,6 +969,65 @@ vim.str_byteindex({str}, {index}, {use_utf16})           *vim.str_byteindex()*
       • {index}      (number)
       • {use_utf16}  any|nil
 
+vim.str_utf_end({str}, {index})                            *vim.str_utf_end()*
+    Returns 0 if the index is pointing at the last byte of a character, 1 if
+    index is pointing at the second last byte of a character, and so on.
```

Comment:
The returned value behaves like a distance or offset. That seems helpful to mention.